docs/source/blocks/block-style-wrapper.md

```diff
@@ -149,6 +149,8 @@ The resultant HTML would be the following:
 ```
 Then it's at your discretion how you define the CSS class names in your theme.
 
+## Customize the injected class names
+
```

Comment:
I don't know in which version this will be released, but this is a helpful admonition for developers. See also https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#admonition-types. There is also `versionchanged` if you think that is more appropriate, but this smells like a new feature, so I went with `versionadded`.

````suggestion

```{versionadded} release-version-number
```

````

Reply:
The classnames injection was released somehwere in middle 16 I think.

Reply:
Oh, I thought you are adding the custom CSS properties feature with this PR?

But for the original version, at the top of the file just below the title, we should have:

````
```{versionadded} 16.0.0
```
````

Reply:
Or are you referring to the interim, where something changed?

https://github.com/plone/volto/commits/5b4ec3f31ed20a12fc3eb25ffc49c9dee11bb4eb/docs/source/blocks/block-style-wrapper.md

If so, then wherever that changed, it should have the same syntax:

````
```{versionchanged} 16.x.x-or-whatever
```
````

Reply:
Moved to the top. The thing added now is only the custom properties, so it's also noted above it with another versionadded.

Reply:
OK. That works for the title, but we need a specific release number for this new feature here. That could be tricky if you backport it, but you would just need to remember to do the chore.

Reply:
Oh, nm, I see you did that below. I had not scrolled down yet.